File: 3 Day/Labs/Interop/Begin/TempConverter/TempConverter/ViewController.m
//
//  ViewController.m
//  TempConverter
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    
    _fResultLabel.text = @"";
    _cResultLabel.text = @"";
    _kResultLabel.text = @"";
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

- (IBAction)convertAction:(id)sender {

    _fResultLabel.text = @"Not implemented";
    _cResultLabel.text = @"Not implemented";
    _kResultLabel.text = @"Not implemented";
    
    [(UIButton *)sender setTitle:@"Convert Again" forState:UIControlStateNormal];
}

@end

File: 2 Day/Labs/Lab07/Begin/TempConverter/TempConverter/ViewController.swift
//
//  ViewController.swift
//  TempConverter
//

import UIKit

class ViewController: UIViewController {

    @IBOutlet weak var tempField: UITextField!
    @IBOutlet weak var resultLabel: UILabel!
    
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }

    @IBAction func convertAction(sender: AnyObject) {
        let temp = Temperature(
            scale:.F, value:(tempField.text! as NSString).doubleValue
        )
        let converter = TempConverter()
        let result = converter.convert(temp, toScale:.C)
        
        resultLabel.text = String(format: "%.2f", result.value)
    }

}

